config/apply/ccsm/translate.js

```diff
@@ -407,81 +399,88 @@ export function translateResponse(customApiResponse, patientData) {
       codings.push(standardTestTypeCodes['Cervical Histology']);
     }
 
-    // Is findingType always provided?
-    let procedureText = procedureMappings[findingType?.ID];
-    let procedureCoding = standardProcedureCodes[procedureText];
+    // Skip add DiagnosticReport if there is no codings created
+    if (codings.length > 0) {
+      const conclusionCodes = [];
```

Comment:
Most tests are now failing because this is `const` rather than `let`, and it gets updated by each of the `mapResults()` calls below.
```suggestion
      let conclusionCodes = [];
```

Reply:
Updated in https://github.com/ccsm-cds-tools/ccsm-cds-service/pull/27/commits/9e264bd967cfb6a6c9b61b151165590f3ca09118, since this is a straightforward syntactical change.

Reply:
Oh, I though conclusionCodes is updated in each mapResults method. You are correct.